### PR TITLE
Upgrade to n-gage and circle v2

### PR DESCRIPTION
**Pull request by liamkeaton**
🐿 v2.12.3

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,167 @@
+# generator: n-circle2-cli
+# template: component
+
+references:
+
+  container_config_node8: &container_config_node8
+    working_directory: ~/project/build
+    docker:
+      - image: circleci/node:8-browsers
```
Comment (sjparkinson) [inline .circleci/config.yml]:
```suggestion
      - image: circleci/node:8.16.0-browsers
```

**Comment by liamkeaton** [comment]
@sjparkinson tested without `-browsers` and it still fails. Is there anything I need to do in the Circle project to shift it to v2? I still get this message on the project although that might be as the master branch hasn't got these changes. 

Screenshot 2019-06-19 at 16 24 14

**Comment by liamkeaton** [comment]
@sjparkinson @Financial-Times/ft-com-conversion all ready to go now!

File: test/session-decoder.test.js [hunk 2]
```diff
@@ -1,38 +1,38 @@
 const expect = require('chai').expect
-    , SessionDecoder = require("../src/session-decoder")
-    , pubKey = "MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAE4U5p7RQ4YR89NPSuQFoNBaras4BSrBirYO1JzAnfVJr9FfEqpDyHiW1Vwg9RcYp7Uo7E_fE6Pq-3rb9lD6m6wA"
-    , decoder = new SessionDecoder(pubKey);
+	; const SessionDecoder = require('../src/session-decoder')
```
Comment (ludrob) [inline test/session-decoder.test.js]: this formatting looks odd to me

Comment (liamkeaton) [reply test/session-decoder.test.js]: Yeah that looks pretty odd, let me fix

**Pull request merged**